src/sentry/api/bases/organization.py

```diff
@@ -103,6 +103,15 @@ class OrganizationIntegrationsPermission(OrganizationPermission):
     }
 
 
+class OrganizationIntegrationsLoosePermission(OrganizationPermission):
+    scope_map = {
+        "GET": ["org:read", "org:write", "org:admin", "org:integrations"],
+        "POST": ["org:read", "org:write", "org:admin", "org:integrations"],
+        "PUT": ["org:read", "org:write", "org:admin", "org:integrations"],
+        "DELETE": ["org:read", "org:write", "org:admin", "org:integrations"],
+    }
```

Comment:
For posterity, the current change of permissions would look like this:
```diff
         "GET":     ["org:read", "org:write", "org:admin", "org:integrations"],
-        "POST":    [            "org:write", "org:admin", "org:integrations"],
+        "POST":    ["org:read", "org:write", "org:admin", "org:integrations"],
-        "PUT":     [            "org:write", "org:admin", "org:integrations"],
+        "PUT":     ["org:read", "org:write", "org:admin", "org:integrations"],
-        "DELETE":  [                         "org:admin", "org:integrations"],
+        "DELETE":  ["org:read", "org:write", "org:admin", "org:integrations"],
```

https://github.com/getsentry/sentry/blob/c605f45a3ca0e585cd7ca1c5a0c286c0d61913e3/src/sentry/api/bases/organization.py#L97-L103

Reply:
As per @NisanthanNanthakumar the member role doesn't have org:write here
https://github.com/getsentry/sentry/blob/f6177828e14fe567b69eea60520cae7262eba39e/src/sentry/conf/server.py#L1652-L1666